src/routines/solve_routines.jl

```diff
@@ -55,9 +55,17 @@ end
 function _run_stage(stage::Stage, start_time::Dates.DateTime, results_path::String; kwargs...)
     @assert stage.internal.psi_container.JuMPmodel.moi_backend.state != MOIU.NO_OPTIMIZER
     timed_log = Dict{Symbol, Any}()
-    _, timed_log[:timed_solve_time],
-    timed_log[:solve_bytes_alloc],
-    timed_log[:sec_in_gc] = @timed JuMP.optimize!(stage.internal.psi_container.JuMPmodel)
+
+    IS.redirect_stdout_to_log() do
+        model = stage.internal.psi_container.JuMPmodel
```

Comment:
is there not a getter function for JuMPmodel?

Reply:
Not yet, needs to be implemented